vassal-doc/src/main/readme-referencemanual/ReferenceManual/Preferences.adoc

```diff
@@ -19,89 +19,100 @@ a <<TurnTracker#top,Turn Tracker>> won't display a Turn Tracker tab). You will f
+*Center on opponents' moves:*:: If checked, whenever an opponent makes a move (either online or while replaying a PBEM log file), VASSAL will ensure that the position is visible on the screen, recentering if necessary. This is generally desirable behavior, but in online games where multiple players make moves simultaneously it can be annoying, and thus the ability to turn it off is provided.
 
 NOTE: Previous versions of VASSAL had a feature which let the value of this preference be forced by a module's settings. This seemed to create problems, so it was removed as of VASSAL 3.4, and now this player preference is always available in every module.
 
-*Reverse left-to-right order of pieces in Stack Viewer*: When checked, the normal left-to-right ordering of pieces in the Mouse-over Stack Viewer is reversed.
+*Reverse left-to-right order of pieces in Stack Viewer:*:: When checked, the normal left-to-right ordering of pieces in the Mouse-over Stack Viewer is reversed.
 
-*Recenter Sensitivity (% of edge/center distance)* Adjusts the sensitivity when VASSAL decides whether to recenter the screen on an action. Larger numbers will produce more recentering.
+*Recenter Sensitivity (% of edge/center distance):*:: Adjusts the sensitivity when VASSAL decides whether to recenter the screen on an action. Larger numbers will produce more recentering.
 
-*Scroll increment (pixels)*: Sets the increment, in mapboard pixels, by which the map scrolls when using its scrollbars.
+*Scroll increment (pixels):*:: Sets the increment, in mapboard pixels, by which the map scrolls when using its scrollbars.
 
-*Use arrow keys to scroll*: If checked, the arrow keys will be mapped to scroll the currently focused Map#top, Map>> window.
+*Use arrow keys to scroll:*:: If checked, the arrow keys will be mapped to scroll the currently focused Map#top, Map>> window.
 
-*Disable automatic stack display - use configured control key (Ctrl+Space) instead*: If checked, Map#StackViewer, Mouseover Stack Viewers>> do not pop up automatically when holding the mouse over a piece or stack. Instead, the viewer appears only when _Ctrl+SPACE_ is then pressed.
+*Disable automatic stack display - use configured control key (Ctrl+Space) instead:*:: If checked, Map#StackViewer, Mouseover Stack Viewers>> do not pop up automatically when holding the mouse over a piece or stack. Instead, the viewer appears only when _Ctrl+SPACE_ is then pressed.
 
-*Delay before automatic stack display (ms)*: Sets the number of milliseconds that the mouse must be held above a piece or stack to activate a Map#StackViewer, Mouseover Stack Viewer>>.
+*Delay before automatic stack display (ms):*:: Sets the number of milliseconds that the mouse must be held above a piece or stack to activate a Map#StackViewer, Mouseover Stack Viewer>>.
```

Comment:
All 3 of these entries are missing the "<<" from their links to the Map document.